src/context.ts

```diff
@@ -73,10 +73,22 @@ export class Context {
   public get issue () {
     const payload = this.payload
 
-    return {
-      ...this.repo,
-      number: (payload.issue || payload.pull_request || payload).number
+    const context: { [k: string]: any } = {
```

Comment:
The variable name `context` is confusing here, because its in the `Context` class - I originally read it as an instance of `Context`. How about:

* issue
* data